### PR TITLE
Packages

### PR DESCRIPTION
**Pull request by andrewdlqnguyen**
Starting from the first sub-branch to build towards master.

At this branch, the master is now separated into 4 packages:
1. Game - To run the game
2. Model - For the datas
3. Controller - handle actionlisteners
4. View

### CODE REVIEW

**Comment by andrewdlqnguyen** [comment]
Updated Issue #14

File: src/controller/testing.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package controller;
+
+public class testing {
```
Comment (vikrambombhi) [inline src/controller/testing.java]: No sure whats the point of a empty testing class

File: src/game/Game.java [hunk 2]
```diff
@@ -1,7 +1,16 @@
+package game;
 import java.util.Date;
 import java.util.Random;
 import java.util.Scanner;
 
+import model.Board;
```
Comment (vikrambombhi) [inline src/game/Game.java]: Can u not just import all of the `model` package at once?

File: src/view/testing.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package view;
+
+public class testing {
```
Comment (vikrambombhi) [inline src/view/testing.java]: Same thing, no point of having a empty testing class

**Review by vikrambombhi — CHANGES_REQUESTED** [review]
Some small changes but this looks good!

**Comment by vikrambombhi** [comment]
Also idk if we need a `game` package because i dont see anything else ever going in there.

File: src/controller/testing.java (hunk 1 above)
Comment (andrewdlqnguyen) [reply src/controller/testing.java]: Only made it to have the folder appear on github. This was removed

Comment (andrewdlqnguyen) [reply src/controller/testing.java]: in later issue

File: src/game/Game.java (hunk 2 above)
Comment (andrewdlqnguyen) [reply src/game/Game.java]: im not sure, that was automatic

File: src/view/testing.java (hunk 3 above)
Comment (andrewdlqnguyen) [reply src/view/testing.java]: deleted at later issue.

**Comment by andrewdlqnguyen** [comment]
okay, can put the game into model. only made it its own package to separate it as a run package

**Comment by vikrambombhi** [comment]
Im actually going to leave it as is because the more I think about it the more I realize idk whats correct. Lets just change it latter if we feel like its wrong.

**Review by vikrambombhi — APPROVED** [review]

**Pull request merged**